albumentations/augmentations/keypoints_utils.py

```diff
@@ -15,6 +16,14 @@
 keypoint_formats = {"xy", "yx", "xya", "xys", "xyas", "xysa"}
 
 
+def angle_to_2pi_range(angle):
+    two_pi = 2 * math.pi
+    if 0 <= angle < two_pi:
```

Comment:
actually, with modulo operator we don't need the check at all and always return `angle % two_pi`